src/client/pydaos/SConscript

```diff
@@ -41,10 +41,18 @@ def build_shim_module(py_env, svars, lib_prefix, version):
                               SHLIBSUFFIX=svars["SO"],
                               CPPFLAGS=svars["BASECFLAGS"] + " " + \
                               svars["OPT"] + " " + extra_flags)
-    install_path = lib_prefix + "/python" + version + "/site-packages"
+    install_path = lib_prefix + "/python" + version + "/site-packages/pydaos"
```

Comment:
We probably need to investigate whether `version` here is aligning with what various distros are dong for their _site-packages_ dirs.